google-cloud-error_reporting/README.md

```diff
@@ -29,7 +29,7 @@ or
 
     $ gcloud auth login
 
-Please see [[gcloud beta auth application-default login][] document for the difference between these commands.
+Please see [gcloud beta auth application-default login][] document for the difference between these commands.
```

Comment:
Yes. The target with empty brackets is so-called 'implicit link', which means that the referenced link can be identified by the link text itself.